Source/WebInspectorUI/UserInterface/Views/NetworkTableContentView.css

```diff
@@ -87,6 +87,26 @@
     cursor: pointer;
 }
 
+body.shift-key-pressed .network-table > .table:hover li:not(:hover, .selected, .initiated) .cell {
+    color: var(--text-color-transparent);
+}
+
+body.shift-key-pressed .network-table > .table:hover li:not(:hover, .selected, .initiated) .cell > * {
+    color: var(--text-color);
+    opacity: 0.4;
+}
```

Comment:
Can we just set the opacity of the entire cell to `40%` and not mess with the `color` at all?

Reply:
sadly no because that will muck with this :(
```
.network-table > .table :not(.header) .cell:first-of-type {
    background: rgba(0, 0, 0, 0.07);
}
```